scripts/apps/desks/services/DesksFactory.js

```diff
@@ -180,7 +180,7 @@ export function DesksFactory($q, api, preferencesService, userList, notify,
                 return $q.when(self.userDesks);
             }
 
-            return this.fetchCurrentDeskId() // make sure there will be current desk
+            return $q.all(this.fetchCurrentDeskId()) // make sure there will be current desk
```

Comment:
I guess this should be `$q.when`

Reply:
yes, I too thought the same, but it does not solve the issue so I used `$q.all`.